### ISSUE TITLE
VTK8 - rendering/pixeldata.py test is crashing in parallel and scalable,parallel,icet modes

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#766]
The rendering/pixeldata.py test is crashing in the parallel and scalable, parallel,icet modes. The tests are in the skip list and should be removed from the skip list when this bug is fixed.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 06/13/2018 12:10 pm
Original update: 06/13/2018 12:14 pm
Ticket number: 3160
